dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -12,6 +12,15 @@ def load_user
     @user = current_user
   end
 
+  # GET /api/v1/users/<user_id>/contact_details
+  def get_contact_details
+    render json: {
+      user_name: current_user.name,
```

Comment:
should we use safe navigation for the name as well? (here and in the mini contact controller)